nms/app/packages/magmalte/app/components/EventAlertChart.js

```diff
@@ -13,17 +13,33 @@
  * @flow strict-local
  * @format
  */
+<<<<<<< HEAD
```

Comment:
whoops

Reply:
surprised, i was able to rebase continue with conflicts. I thought I had saved and added the file.